Resources/Textures/Corvax/Back/Backpacks/ce.rsi/meta.json

```
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "JerryTheMouse#8561",
```

```suggestion
  "copyright": "Discord: JerryTheMouse#8561",
```
Неясно о чём именно речь, хотя бы так, и в остальных укажи.

---

Кхм, да хрен с ним, я уже устал от этого ПРа